src/client/scripts/cardGrid.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+/*global Cards:false Rooms:false */
+
+Template.cardGrid.onRendered(function(){
+    Session.set("getUniqueID_CallCount",0);
+    Session.set("pairSet",false);
+});
+
+Template.cardGrid.helpers({
+
+    validMultiRow: function(){
+        var categories = Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+        var len = categories.length;
+
+        return len > 4;
+    },
+
+    getRow: function(){
+        var rowOfCategories = [];
+        var categories = Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+        var index = 0;
+        var numInnerArrays = Math.floor(4 / categories.length);
+
+        numInnerArrays += 4 % categories.length;
+        /* creates an array of subarray
+        *  each subarray holds category
+        *  im using this technique to creates
+        *  a more bootstrap grid view
+        */
+        for (var i = 0; i < numInnerArrays; i++) {
+            rowOfCategories[i] = new Array();
+        }
+        for (var j = 0; j < categories.length; index++) {
+            for (var k = 0; k < 4 && j < categories.length; k++) {
+                rowOfCategories[index][k] = categories[j];
+                j++;
+            }
+        }
+        return rowOfCategories;
+    },
+
+    getCategories: function() {
+        return Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+    },
+
+    getNumCategories: function(){
+        var cat = Rooms.findOne(
+          {"roomCode": Session.get("roomNumber")}
+      ).categories;
+
+        return Math.floor(12 / cat.length);
```

Comment:
why is this not returning cat.length?  If the method is called getNumCategories it should return the number of categories, nothing else.  Do your Math.floor after the method is called.